src/Pages/CommonComponents/CommonPage.js

```diff
@@ -1,8 +1,72 @@
-import React from "react";
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./CommonPage.scss";
 
-class CommonPage extends React.Component {
+class CommonPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      profilesTop: [
+        {
+          goTo: "/login-yeheum",
+          alt: "yeheum",
+          src: "/images/yeheum/me.jpeg",
+        },
+        {
+          goTo: "/login-tae",
+          alt: "taegyu",
+          src: "/images/yeheum/tae.jpg",
+        },
+      ],
+      profilesBottom: [
+        {
+          goTo: "/login-oh",
+          alt: "seungha",
+          src: "/images/yeheum/seungha.jpg",
+        },
+        {
+          goTo: "/login-gy",
+          alt: "gyuyoung",
+          src: "/images/yeheum/dog.png",
+        },
+      ],
+    };
+  }
   render() {
-    return <></>;
+    const { profilesTop, profilesBottom } = this.state;
+    return (
+      <section className="CommonPage">
+        <div className="WrapMainLogo">
```

Comment:
```suggestion
        <div className="wrapMainLogo">
```
이전 리뷰 내용에 있었던 것 같은데 빠짐 없이 반영해주세요. 대문자로 시작하는 건 컴포넌트에서 최상위 태그의 클래스명 뿐입니다.